controllers/storage.go

```diff
@@ -257,3 +278,25 @@ func (r *DSPAReconciler) ReconcileStorage(ctx context.Context, dsp *dspav1alpha1
 
 	return nil
 }
+
+func (r *DSPAReconciler) retrieveAndSetExternalRoute(ctx context.Context, dsp *dspav1alpha1.DataSciencePipelinesApplication, params *DSPAParams, client client.Client) error {
+
+	log := r.Log.WithValues("namespace", dsp.Namespace).WithValues("dspa_name", dsp.Name)
+	// Retrieve the external route
+	route := &routev1.Route{}
+	namespacedName := types.NamespacedName{
+		Name:      "minio-" + params.Name,
+		Namespace: params.Namespace,
+	}
+	if err := client.Get(ctx, namespacedName, route); err != nil {
+		log.Info("Unable to retrieve route", "error", err)
```

Comment:
imho, should be log.Error

Reply:
@diegolovison - log.Error should be used very sparingly as it will stop the cause the entire controller to fail and stop.  I don't believe this is one such case